types/setting.go

```diff
@@ -1041,3 +1041,13 @@ func UnmarshalNodeSelector(nodeSelectorSetting string) (map[string]string, error
 	}
 	return nodeSelector, nil
 }
+
+var disableProfiler bool = false
+
+func SetProfilerDisabled(value bool) {
+	disableProfiler = value
+}
+
+func IsProfilerDisabled() bool {
+	return disableProfiler
+}
```

Comment:
We don't need this global variable for now, so please remove them first. Thanks.

Reply:
Removed.